src/plugins/plot/src/telemetry/PlotController.js

```diff
@@ -277,22 +277,18 @@ define([
      * Export view as JPG.
      */
     PlotController.prototype.exportJPG = function () {
-        this.hideExportButtons = true;
-        this.exportImageService.exportJPG(this.$element[0], 'plot.jpg', 'export-plot')
-            .finally(function () {
-                this.hideExportButtons = false;
-            }.bind(this));
+        var plotElement = this.$element.children()[1];
+
+        this.exportImageService.exportJPG(plotElement, 'plot.jpg', 'export-plot');
```

Comment:
Does this still export the plot legend?

Reply:
Yes, the only div its not capturing is 
` <div class="c-control-bar">`